ui/src/app/tasks-jobs/tasks/launch/builder/builder.component.ts

```diff
@@ -1118,6 +1120,14 @@ export class BuilderComponent implements OnInit, OnDestroy {
     }
   }
 
+  getLabelAppVersion(taskLaunchConfig, app) {
+    const lastVersion = get(taskLaunchConfig?.lastExecution?.deploymentProperties, `version.${app.name}`);
+    if (lastVersion && app.version !== lastVersion) {
+      return `Last launch (${lastVersion})`;
```

Comment:
Maybe it would be nice to integrate Angular i18n instead of using fixed english Strings: https://angular.io/guide/i18n-overview